climakitae/indices.py

```diff
@@ -4,6 +4,40 @@
 import numpy as np
 
 
+def effective_temp(T):
```

Comment:
Based on my understanding of this metric, I think the "yesterday" temp needs to be the "yesterday effective" temp, which confirms that yes the first time step will be nan, but to me it looks like the T_yesterday is just grabbing yesterday's temperature _not_ yesterday's effective temperature, if that makes sense

Reply:
Hmmm that's not actually how I understood the equation. I wasn't able to find any additional documentation on the metric though. It's described in the [reference doc](https://www.nationalgas.com/document/132516/download) Owen sent me this way: 
> The effective temperature is half of yesterday’s temperature added to half of today’s actual
temperature.

Reply:
Ooooo you're right! Good catch. I'll update it now
From the reference doc: 
Effective temperature (0.5 * today’s temperature + 0.5 * yesterday’s effective)